server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/manager.js

```diff
@@ -25,29 +25,52 @@
 
 var managerStatusTable, recoveryListTable;
 
-function refreshManagerBanner() {
+function refreshManagerBanners() {
   getStatus().then(function () {
-    var managerStatus = JSON.parse(sessionStorage.status).managerStatus;
+    const managerStatus = JSON.parse(sessionStorage.status).managerStatus;
 
     // If manager status is error
     if (managerStatus === 'ERROR') {
-      // show banner and hide table
-      $('#managerBanner').show();
+      // show the manager error banner and hide table
+      $('#managerRunningBanner').show();
       $('#managerStatus_wrapper').hide();
     } else {
-      // otherwise, hide banner and show table
-      $('#managerBanner').hide();
+      // otherwise, hide the error banner and show manager table
+      $('#managerRunningBanner').hide();
       $('#managerStatus_wrapper').show();
     }
   });
+
+  getManager().then(function () {
+    const managerData = JSON.parse(sessionStorage.manager);
+    const managerState = managerData.managerState;
+    const managerGoalState = managerData.managerGoalState;
+
+    const isStateGoalDifferent = managerState !== managerGoalState;
```

Comment:
> I think some of the logic might read better if there were fewer negations.

I inverted this boolean in a new commit.
 
> Also, what happens if these are the same, but both are null?

The banner will be hidden when `managerState === null` is checked.

> What happens if `getManager()` errors?

We don't have much error handling anywhere in the monitor code but I could add a catch block in this case to `console.error` the error.

Reply:
> We don't have much error handling anywhere in the monitor code but I could add a catch block in this case to `console.error` the error.

If there's an error, I think it would show up on the console anyway. I was mostly thinking about making sure the rendered view was sensible in those edge cases. I think if it errors, then it will keep whatever the current message is, whether it's hidden or shown on the page. That could lead to user confusion if it contradicts the message in the other banner about the manager not running.

Reply:
I think we don't need to worry about this edge case too much unless we start seeing it a lot.